daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -328,6 +328,20 @@ abstract class AbstractFuncIT
         }
       }
     }
+    "ExceptionAndContractKey:test" should {
```

Comment:
is there a strong reason why this should be a daml script rather than a ledger API test tool test?

Currently, Daml script tests test that Daml Script functions properly. Here you are really testing that the ledger (or the engine in the ledger) functions correctly.

Reply:
move to unit test in engine module. 
fc57a8f6b40886821530bcb695e23a3777d31167